### PR TITLE
docs: Add settings and commands section

### PR DESCRIPTION
**Pull request by xiyaowong**
Inform users to find all available settings and commands in the Features tab of the extension details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +116,11 @@ versions for the best experience. These can be found
     `defaults write com.microsoft.VSCode ApplePressAndHoldEnabled -bool false`.
 -   To fix the remapped escape key not working in Linux, set `"keyboard.dispatch": "keyCode"`
 
+### Settings & Commands
+
+Please open the [vscode-neovim](vscode:extension/asvetliakov.vscode-neovim) extension details in VSCode, and check all
+available [settings](https://code.visualstudio.com/docs/getstarted/settings) and commands in the Features tab.
```
Comment (ollien) [inline README.md:122]: I don't think this link is necessary

Comment (xiyaowong) [reply README.md:122]: Which one? (Please do not trust your user.) I referred to the documentation of other extensions, such as vscode-eslint.

File: README.md [hunk 2]
```diff
@@ -115,6 +116,11 @@ versions for the best experience. These can be found
     `defaults write com.microsoft.VSCode ApplePressAndHoldEnabled -bool false`.
 -   To fix the remapped escape key not working in Linux, set `"keyboard.dispatch": "keyCode"`
 
+### Settings & Commands
+
+Please open the [vscode-neovim](vscode:extension/asvetliakov.vscode-neovim) extension details in VSCode, and check all
```
Comment (ollien) [inline README.md:121]: Nit: this is phrased like a request, when really it's just informational

File: README.md (hunk 1 above)
Comment (ollien) [reply README.md:122]: The one to the official docs. Given the number of places we refer to various settings in this README, it seems strange to presume they don't know VSCode can be configured

File: README.md (hunk 2 above)
Comment (xiyaowong) [reply README.md:121]: Not good at writing in English

Comment (ollien) [reply README.md:121]: That's ok. Let me try to craft something

Comment (xiyaowong) [reply README.md]: Changed

File: README.md (hunk 1 above)
Comment (xiyaowong) [reply README.md:122]: :joy:

File: README.md (hunk 2 above)
Comment (xiyaowong) [reply README.md]: > That's ok. Let me try to craft something

Feel free to modify this PR, I have no objection.

**Comment by xiyaowong** [comment]
@ollien @theol0403  Do you have any suggestions for changes? I would like to merge this PR into the next minor release.

**Comment by ollien** [comment]
Sorry, I meant to do this last night but ended up working late. I'll take a closer look tonight

**Comment by theol0403** [comment]
Put any review comments in https://github.com/vscode-neovim/vscode-neovim/pull/2155

**Pull request merged**